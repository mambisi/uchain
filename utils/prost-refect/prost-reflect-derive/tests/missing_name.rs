use prost_reflect_derive::ReflectMessage;

#[derive(ReflectMessage)]
#[prost_reflect(descriptor_pool = "DESCRIPTOR_POOL")]
pub struct MyMessage {}

fn main() {}
