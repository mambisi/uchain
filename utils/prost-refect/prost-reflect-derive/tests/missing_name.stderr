error: missing required argument 'message_name'
 --> tests/missing_name.rs:4:1
  |
4 | #[prost_reflect(descriptor_pool = "DESCRIPTOR_POOL")]
  | ^
